website/docs/r/vapp.html.markdown

```diff
@@ -24,6 +24,12 @@ resource "vcd_network_direct" "net" {
 resource "vcd_vapp" "web" {
   name = "web"
 
+  metadata {
+    role    = "web"
```

Comment:
Would be nice to add a comment that any keys are supported and to make a point possibly use different key names across the three samples, because a newcomer may think "are these keys the only ones I can use or may I use anything"?

Reply:
Fixed. I hope you like it. :)

Reply:
Most of them, but some could be toned down ;)